packages/neon-core/__integration__/rpc/clients/NeoServerRpcClient.ts

```diff
@@ -281,14 +281,13 @@ describe("NeoServerRpcClient", () => {
         ]
       );
 
-      expect(Object.keys(result)).toHaveLength(6);
```

Comment:
This was my mistake. I copied the integration docker config to my local single node so I could debug the C# side. I didn't realise the `tx` attribute was only added when adding signers _and_ having a wallet open (which I had open while running my node)